src/tikzdocument.jl

```diff
@@ -194,6 +194,9 @@ an existing PDF file with the same name.
                      latex_engine = latexengine(),
                      buildflags = vcat(DEFAULT_FLAGS, CUSTOM_FLAGS),
                      keep_pdf = false)
+        if _is_ijulia() || _is_juno() || _is_vscode()
+
```

Comment:
what does this do? it looks like an empty `if ... end` block

Reply:
Oh, I think this was left on a WIP to make the background white if we are in an IDE. Will remove it.